tests/instruments/keysight/test_keysight35670A_with_device.py

```
@@ -0,0 +1,341 @@
+
+
+@pytest.fixture(autouse=True)
+def ensure_safe_state(analyzer, request):
+    """Force output OFF and fail clearly when any post-test instrument error remains."""
+    analyzer.source_output_enabled = False
+    _assert_no_system_error(analyzer, "ensure_safe_state", max_reads=32)
+    yield
+    notes = []
+    try:
+        analyzer.source_output_enabled = False
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)